Taarafo.Core/Models/PostImpressions/Exceptions/NullPostImpressionException.cs

```diff
@@ -0,0 +1,16 @@
+// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using Xeptions;
+
+namespace Taarafo.Core.Models.PostImpressions.Exceptions
+{
+    public class NullPostImpressionException : Xeption
+    {
+        public NullPostImpressionException()
+            : base(message: "Post Impression is null.")
```

Comment:
Some messages have `Post Impression` others have `Post impression`. Please, update messages to be consistent. I like `Post impression`...

Reply:
I updated the messages in CODERUB: Updated Messages For Post Impression Exceptions. Pull Request link is here: https://github.com/hassanhabib/Taarafo.Core/pull/221